friends2/src/App.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Route } from 'react-router-dom';
+import styled from 'styled-components';
+
+import logo from './logo.svg';
+import './App.css';
+import FriendsList from './FriendsList';
+
+class App extends Component {
+  constructor(props){
+    super(props)
+    this.state ={
+      friends: [],
+    }
+  }
+  
+  componentDidMount() {
+    axios
+      .get("http://localhost:5000/friends")
+      .then( response => {
+        this.setState({friends: response.data})
+        console.log(this.state.friends)
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
console.error would make this a little more apparent to differentiate from logs.